src/components/OcBreadcrumb.vue

```diff
@@ -158,14 +168,26 @@ export default {
 
     .uk-drop > .uk-card > .uk-nav-default {
       > li a,
+      > li button,
       > li span {
         color: var(--oc-color-text-muted);
+        font-size: 0.875rem;
       }
 
       > li a:hover,
       > li span:hover {
         color: var(--oc-color-swatch-brand-default);
       }
+
+      > li a:focus {
+        outline: -webkit-focus-ring-color auto 1px;
```

Comment:
@pascalwengerter I think we need to move this to oc-drop, right?

Reply:
Could you try moving everything inside `&-drop {` there and see how hard it is to make it work? :D

Reply:
https://github.com/owncloud/owncloud-design-system/pull/1230 
done here

Reply:
@pascalwengerter if #1230 finally works and get merged I will remove this line here

Reply:
removed